### PR TITLE
Feed

### PR DESCRIPTION
**Pull request by A-n-u-O**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -34,6 +38,26 @@ function App() {
     {
       path: "Dashboard",
       element: <Dashboard />,
+      children: [
+        {
+          path: "Home",
```
Comment (Boye-dev) [inline src/App.tsx:43]: Put your route names in lowercase

File: src/App.tsx [hunk 2]
```diff
@@ -34,6 +38,26 @@ function App() {
...
+          element: <Home />,
+        },
+        {
+          path: "Feed",
```
Comment (Boye-dev) [inline src/App.tsx:47]: Put your route names in lowercase

File: src/App.tsx [hunk 3]
```diff
@@ -34,6 +38,26 @@ function App() {
...
+          element: <Feed />,
+        },
+        {
+          path: "Messages",
```
Comment (Boye-dev) [inline src/App.tsx:51]: Put your route names in lowercase

File: src/App.tsx [hunk 4]
```diff
@@ -34,6 +38,26 @@ function App() {
...
+          element: <Messages />,
+          children: [
+            {
+              path: "ChatBox",
```
Comment (Boye-dev) [inline src/App.tsx:55]: You do not need to add another route for this chatbox just add it tto the messages component directly

**Review by Boye-dev — COMMENTED** [review]
Add a screenshot of what youve done

File: src/App.tsx (hunk 1 above)
Comment (A-n-u-O) [reply src/App.tsx:43]: okay

File: src/App.tsx (hunk 4 above)
Comment (A-n-u-O) [reply src/App.tsx:55]: okay

**Pull request merged**